src/main/webapp/components/BuildSnapshotContainer.js

```diff
@@ -43,10 +52,27 @@ export const BuildSnapshotContainer = React.memo((props) =>
           No new revisions to display as of {new Date().toLocaleString()}.</span>;
     }
 
+    let hasPrev = pageNumber == 0;
```

Comment:
Use <code>===</code> for [equality checks](https://google.github.io/styleguide/jsguide.html#features-equality-checks)
```suggestion
    let hasPrev = pageNumber === 0;
```